packages/app/app/actions/importfavs.js

```diff
@@ -0,0 +1,99 @@
+import logger from 'electron-timber';
+import { LastFmApi } from '@nuclear/core';
+import { store } from '../persistence/store';
+import globals from '../globals';
+import * as FavoritesActions from './favorites';
+const lastfm = new LastFmApi(globals.lastfmApiKey, globals.lastfmApiSecret);
+
+export const FAV_IMPORT_INIT = 'FAV_IMPORT_INIT';
+
+export const LASTFM_FAV_IMPORT_START = 'LASTFM_FAV_IMPORT_START';
+export const LASTFM_FAV_IMPORT_SUCCESS_1 = 'LASTFM_FAV_IMPORT_SUCCESS_1';
+export const LASTFM_FAV_IMPORT_SUCCESS_FINAL = 'LASTFM_FAV_IMPORT_SUCCESS_FINAL';
+export const LASTFM_FAV_IMPORT_ERROR = 'LASTFM_FAV_IMPORT_ERROR';
+
+export function FavImportInit() {
```

Comment:
If you only want one action, no need to use dispatch, just return the action object directly.